src/telegrafs/components/TelegrafConfig.tsx

```diff
@@ -36,7 +33,7 @@ export class TelegrafConfig extends PureComponent<Props> {
         <Editor
           script={config}
           onChangeScript={onChangeConfig}
-          readOnly={isFlagEnabled('editTelegrafs') === false}
+          readOnly={false}
```

Comment:
i think this should be true

Reply:
but `true === false` will return `false`